app/assets/javascripts/components/restaurant.js.jsx

```diff
@@ -1,24 +1,43 @@
 class Restaurant extends React.Component {
   constructor(props){
     super(props);
-
     this.state = {
       restaurant: props.restaurant
     }
   }
-  render(){
+  //I don't like doing it like this, but this is a quick solution for the problem
+  //In a future iteration I will abstract cuisine as its own object that will
+  //be composed inside the restaurant object
+  TransformCuisineToFontLetter(cuisine) {
+    if(cuisine === 0){
+      return 'a';
+    } else if(cuisine === 1){
+      return 'g';
+    } else if(cuisine === 2){
+      return '@';
+    }
+  }
+  CreateStarString() {
+    var stars = "";
+    for (var i=0; i<this.state.restaurant.rating ; i++) {
+      stars += "⭐";//This string may seem empty in some encodings. It has a star symbol.
```

Comment:
I think you can store this as "\u2b50"

http://www.charbase.com/2b50-unicode-white-medium-star